observations/obs_converters/README.rst

```diff
@@ -421,11 +413,20 @@ For generating synthetic observations, see the
 `create_obs_sequence <../../assimilation_code/programs/create_obs_sequence/create_obs_sequence.html>`__
 program documentation. You can also generate observation files based on
 text input. See the `text_to_obs <text/text_to_obs.html>`__ program
-documentation. Or for simulating a large complex observing system, you
+documentation and even_sphere. Or for simulating a large complex observing system, you
```

Comment:
i don't think these two are similar enough to mention here.  text_to_obs takes something like CSV data from excel and outputs an obs_seq file.  even_sphere outputs text specifically for input to create_obs_sequence.  i'd revert this change.